src/WebJobs.Extensions.DurableTask.Analyzers/SyntaxNodeUtils.cs

```diff
@@ -113,7 +107,7 @@ internal static bool TryGetMethodReturnTypeNode(SyntaxNode node, out SyntaxNode
 
         private static bool TryGetChildTypeNode(SyntaxNode node, out SyntaxNode childTypeNode)
         {
-            childTypeNode = node.ChildNodes().Where(x => x.IsKind(SyntaxKind.IdentifierName) || x.IsKind(SyntaxKind.PredefinedType) || x.IsKind(SyntaxKind.GenericName) || x.IsKind(SyntaxKind.ArrayType) || x.IsKind(SyntaxKind.TupleType)).FirstOrDefault();
+            childTypeNode = node.ChildNodes().Where(x => x.IsKind(SyntaxKind.IdentifierName) || x.IsKind(SyntaxKind.PredefinedType) || x.IsKind(SyntaxKind.GenericName) || x.IsKind(SyntaxKind.ArrayType) || x.IsKind(SyntaxKind.TupleType) || x.IsKind(SyntaxKind.NullableType)).FirstOrDefault();
```

Comment:
One line is starting to get hard to read.

```suggestion
            childTypeNode = node.ChildNodes()
                                .Where(
                                    x => x.IsKind(SyntaxKind.IdentifierName) 
                                    || x.IsKind(SyntaxKind.PredefinedType)
                                    || x.IsKind(SyntaxKind.GenericName)
                                    || x.IsKind(SyntaxKind.ArrayType)
                                    || x.IsKind(SyntaxKind.TupleType)
                                    || x.IsKind(SyntaxKind.NullableType))
                                .FirstOrDefault();
```